info.csv

```diff
@@ -53,3 +53,4 @@ Ashwani Tripathi,        18SCSE1140015,          1801140063,            Android,
 Piyush Kumar,            18SCSE1120015,          18021120014,           Machine Learning,               1,            9,        C++,                          "CLOUD COMPUTING",        NA,                   pkumar22122001@gmail.com,      `   https://github.com/kpiyush04,        https://www.linkedin.com/in/piyush-kumar-561309156/ ,    living on world of clouds
 NIKHIL YADAV,18SCSE1010287,18021011523,ANDROID DEVELOPMENT,1,1,PYTHON JAVA,APP DEVELOPMENT,NA,YNIKHIL95@GMAIL.COM,https://github.com/nikhil-ydv,https://www.linkedin.com/in/nikhil-yadav-788981175,NA
 Aryan,                   18SCSE1010395,          18021011626,           Cybersecurity,                  1,            2,      "Java,C,Python,IOT,Cybersecurity","Gaming,PC Building and Modding,Custom ROM Development","Smart-Mirror,Smart-Speaker,IOT-projects",3ncry9t3d@gmail.com,www.github.com/SKYR4,www.linkedin.com/skyr4,"In this world of binary,I am 0 and u can be my 1"
+Ashutosh Rai,            17SCSE101205,           1713101194,            ANDROID DEVELOPMENT,            2,            14,       JAVA,C,PYTHON,CPP,             MOBILE APP DDEVELOPMENT, WEB APP DEVELOPMENT, IOT,  NA, styamrai436@gmail.com     https://github.com/Ashutosh436        https://www.linkedin.com/in/ashutosh-rai-486301159/,         NA                                                               
```

Comment:
Wrap multi-valued cols inside quotes!
like `JAVA,C,PYTHON,CPP` should be `"JAVA,C,PYTHON,CPP"`